docker_images/ci-functional/Dockerfile

```diff
@@ -87,14 +87,25 @@ RUN wget https://dl.google.com/android/repository/android-ndk-r23c-linux.zip &&
     unzip android-ndk-r23c-linux.zip -d /opt/ && \
     rm android-ndk-r23c-linux.zip
 
-ENV BAZEL_VERSION 6.5.0
+# Bazel installation
+ENV BAZEL_6_3_2 6.3.2
+ENV BAZEL_7_1_2 7.1.2
 
 RUN apt install -y apt-transport-https curl gnupg \
-    && curl -LO https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh \
-    && chmod +x bazel-$BAZEL_VERSION-installer-linux-x86_64.sh \
-    && ./bazel-$BAZEL_VERSION-installer-linux-x86_64.sh \
-    && rm bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
-
+    # Bazel 6.3.2
+    && curl -LO https://github.com/bazelbuild/bazel/releases/download/$BAZEL_6_3_2/bazel-$BAZEL_6_3_2-linux-x86_64 \
+    && chmod 755 bazel-$BAZEL_6_3_2-linux-x86_64 \
+    && mv bazel-$BAZEL_6_3_2-linux-x86_64 bazel \
+    && mkdir -p /usr/share/bazel-$BAZEL_6_3_2/bin \
+    && mv bazel /usr/share/bazel-$BAZEL_6_3_2/bin/ \
```

Comment:
This will not be in PATH, I guess you are expecting it already, but how people will use it, I mean, you will need to update the tests to invoke bazel using this new folder path, right?

Reply:
yes, exactly, I'm expecting that

Reply:
@uilianries UPDATE:
See this PR https://github.com/conan-io/conan/pull/16196/commits/5163c77021ed9bf532a280b2c0827fb5f6b969ff

Reply:
Perfect!